qa-test

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 set -e
 
-find . -name '*.py' | grep -v share/python | grep -v zabbix | xargs -r pylint -E --disable=C,R,undefined-variable  --persistent=n --init-hook="sys.path[0:0] = ['./centreon/share/python']"
+find . -name '*.py' | xargs -r grep -L "#!/usr/bin/python3" | grep -v share/python | grep -v zabbix | xargs -r pylint -E --disable=C,R,undefined-variable  --persistent=n --init-hook="sys.path[0:0] = ['./centreon/share/python']"
+find . -name '*.py' | xargs -r grep -l "#\!/usr/bin/python3" | grep -v share/python | grep -v zabbix | xargs -r pylint3 -E --disable=C,R,undefined-variable  --persistent=n
```

Comment:
why a grep "#\!" and a grep "#!"

Reply:
well spotted, should be both with \ since in some cases, the shebang match can be interpreted